manifests/o/OliverSchwendener/ueli/9.25.0/OliverSchwendener.ueli.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created with komac v2.12.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: OliverSchwendener.ueli
+PackageVersion: 9.25.0
+PackageLocale: en-US
+Publisher: Oliver Schwendener
+PublisherUrl: https://github.com/oliverschwendener/ueli
+PublisherSupportUrl: https://github.com/oliverschwendener/ueli/issues
+Author: Oliver Schwendener
+PackageName: ueli
+PackageUrl: https://www.ueli.app/
+License: MIT
+LicenseUrl: https://github.com/oliverschwendener/ueli/blob/HEAD/LICENSE

```

```suggestion
LicenseUrl: https://github.com/oliverschwendener/ueli/blob/main/LICENSE
```

I prefer to use the license on the branch "main", which is also the same branch where `CopyrightUrl` is located.